pkg/printers/internalversion/printers.go

```diff
@@ -41,6 +41,7 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/kubernetes/cmd/kubeadm/app/constants"
```

Comment:
This is a very painful user-facing regression.  Fixing the user-facing regression should take priority over package-bickering.

I agree it's not a good place for it, but by putting it somewhere where everyone can agree it shouldn't be then we're more likely to get a faster resolution, and it's less likely to end up just staying where it is :-)